src/main/kotlin/com/terraformation/backend/accelerator/db/ReportStore.kt

```diff
@@ -1334,6 +1339,47 @@ class ReportStore(
           )
           .convertFrom { results -> results.map { ReportProjectMetricModel.of(it) } }
 
+  private fun userIsInSameOrg() =
+      with(ORGANIZATION_USERS) {
+        DSL.field(
```

Comment:
Couldn't initially think of a better way to do this field, but open to ideas as it doesn't seem very clean (e.g. `systemUser` can't enumerate `organizationRoles` so it needs a separate condition

Reply:
For the system user, this could always be `DSL.falseCondition()`, I think.